File: comms/push-notifications/device-registrations/list-device-registrations/list-device-registrations.py
from twilio_comms import TwilioComms

client = TwilioComms(
    account_id="TWILIO_ACCOUNT_SID",
    auth_token="TWILIO_AUTH_TOKEN",
)

client.push_notifications.device_registrations.list(
    user_id="comms_pushnotificationuser_01h9krwprkeee8fzqspvwy6nq8",
    app_name="appName",
    page_token="pageToken",
    page_size=50,
)

File: comms/push-notifications/device-registrations/list-device-registrations/list-device-registrations.php
<?php

namespace Example;

use Twilio\Comms\TwilioComms;
use Twilio\Comms\PushNotifications\DeviceRegistrations\Requests\DeviceRegistrationsListRequest;

$client = new TwilioComms(
    accountId: 'TWILIO_ACCOUNT_SID',
    authToken: 'TWILIO_AUTH_TOKEN',
);
$client->pushNotifications->deviceRegistrations->list(
    new DeviceRegistrationsListRequest([
        'userId' => 'comms_pushnotificationuser_01h9krwprkeee8fzqspvwy6nq8',
        'appName' => 'appName',
        'pageToken' => 'pageToken',
        'pageSize' => 50,
    ]),
);
